src/Silverstripe/Pingdom/Command/Checks/Get.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Silverstripe\Pingdom\Command\Checks;
+
+use Silverstripe\Pingdom\Api\Client;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Get extends Command
+{
+    // the name of the command (the part after "bin/console")
+    protected static $defaultName = 'checks:get';
+    private $pingdomToken;
+
+    protected function configure()
+    {
+        $this->setDescription('get all pingdom checks attached to the provided account token')
+        ->addOption('json')
+        ->addArgument('api-key', InputArgument::OPTIONAL, 'your oauth token', '');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if (strlen($input->getArgument('api-key')) > 0) {
```

Comment:
```suggestion
        if (count($input->getArgument('api-key')) > 0) {
```

Reply:
dang first google result proves wrong
thought it was count() :smile: